docs/RBC_EN.md

```diff
@@ -0,0 +1,52 @@
+# RBC : Reliable Broadcast
+
+
+
+### overview
+
+The typical Broadcast, which depends on implementation, does not guarantee message delivery and does not consider situations in which the sender is crash (the situation in which the message cannot be sent to all). However, the RBC guarantees message delivery and partially solves the situation where the sender crashes. And the Atomic Broadcast introduced in this [paper](<https://eprint.iacr.org/2016/199.pdf>) is a Broadcast that guarantees the situation of the sender crash and guarantees the order of messages, and the RBC to be introduced is Atomic Broadcast.
+
+
+
+[The paper](<https://eprint.iacr.org/2016/199.pdf>) shows ABC (Atomic Broadcast) as ACS (Asynchronous Common Subset) using `threshold encryption`and ACS consists of RBC and BBA (Binary Byzantine Agreement) components. Among them, RBC performs reliable broadcasting and BBA is responsible for ordering of consensus and atomic messages.
```

Comment:
Typo! Plz space between `threshold encryption` and `and`.

Reply:
In `responsible for ordering of consensus and atomic messages`, `ordering of consensus` is confused to me. Did you intend `ordering of consensus` **meaning a sequence of consensus**, not **consensus for ordering of atomic messages**?

Reply:
sequence of consensus!